addons/no-sprite-confirm/addon.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Delete sprites without confirmation",
+  "description": "Removes the confirmation prompt that asks if you are sure you want to delete a project sprite.",
```

Comment:
```suggestion
  "description": "Removes the confirmation prompt that asks if you are sure you want to delete a sprite.",
```

Reply:
```suggestion
  "description": "Removes the confirmation prompt that asks if you are sure you want to delete a sprite in a project.",
```

Reply:
It could also say "delete a sprite from a project". But I don't think there's a reason to mention projects - "sprite" can't mean anything that isn't inside a project.